app/services/activatable/activation_service.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module Activatable
```

Comment:
Bu isim düzenlemesine göre `Activation::ActivationService` yerine `Activation::WhateverActivationService`'e oynuyoruz demektir.  Nitekim koda baktığımda bu "Whatever"in "Student" olduğunu görüyorum ("ProspectiveStudent"tan dolayı).  Özetle `s/activation_service/student_activation_service/`, `s/ActivationService/StudentActivationService` oluyor.  Bu düzenleme gelecekte sahneye girecek olan "Employee"nin de yerini hazırlayacaktır.  Ha o gün geldiğinde küçük bir refaktör gerekebilir[1], ama bunu o güne bırakalım ve hazırlığı bununla sınırlı tutalım.

[1] Muhtemel refaktör şöyle olacak.  `StudentActivationService` ve `EmployeeActivationService` ortaklığı `User` olduğuna göre belki bir `UserActivationService` mixin'i veya concern'i açıp ortaklığı o modülde yöneteceksin.  Bunu şimdiden kestirmem zor.  Kestirime dayalı bir aksiyonda bulunmak doğru da değil (YAGNI).